.eslintrc

```diff
@@ -1,3 +1,26 @@
 {
-    "extends": "@apify/eslint-config-ts"
-}
+    "extends": [
+      "@apify/ts"
+    ],
+    "parserOptions": {
+      "project": "tsconfig.eslint.json"
+    },
+    "overrides": [
+      {
+        "files": [
+          "src/**/*.ts"
+        ],
+        "rules": {
+          "import/extensions": 0
+        }
```

Comment:
This could be added to the eslint-config, no? I think I'm seeing this repeatedly.

Reply:
I just copied everything from `browser-pool`.

Reply:
Yeah, I know.

Reply:
Pretty sure it could be moved there, yes